src/content/blog/whats-new-august-2024.mdx

```diff
@@ -0,0 +1,176 @@
+- [`stripe-astro-loader`](https://github.com/notrab/stripe-astro-loader) - Fetch data from the Stripe API and use it in Astro collections.
+- [Chuck Norris facts content loader](https://gist.github.com/NuroDev/4957f92f957d6c26fff216cff4c082fa) - You knew someone was going to do it!
+- [`@jcayzac/astro-build-cache`](https://www.npmjs.com/package/@jcayzac/astro-build-cache) - A simple build cache for artifacts you may be generating during an Astro build.
+- [`@jcayzac/astro-classlist`](https://www.npmjs.com/package/@jcayzac/astro-classlist) - A small utility to coerce/sanitize class:list when you want to pass it to a child component.
+- [`notion-astro-loader`](https://github.com/NotWoods/travel/tree/main/packages/notion-astro-loader) - Load pages from a Notion database then render them as pages in a collection.
+- [DatoCMS Astro starter kit](https://www.datocms.com/marketplace/starters/astro-starter-kit)
+- [`astro-loader-gather`](https://github.com/dustinlacewell/astro-loader-gather) - A loader for Astro's Content Layer similar to the built-in glob loader with some extensions for extracting metadata from filesystem paths.
+- [`mdxtree`](https://github.com/mayneyao/mdxtree) - A Rust command-line tool that generates a file tree structure in MDX format for use with Astro's Starlight components.
+- [`astro-social-share`](https://www.npmjs.com/package/astro-social-share) - v2.0! Social media share buttons for your Astro site.
+- [`astro-dump`](https://www.npmjs.com/package/astro-dump) - A better debugging experience.
+- [`astro-console-cleaner`](https://www.npmjs.com/package/astro-console-cleaner) - An Astro integration that removes `console.log`, `console.warn`, and `console.error` statements from your code during the build process.
+- [Astro Breadcrumbs](https://docs.astro-breadcrumbs.kasimir.dev/) - A configurable breadcrumb component for Astro.
+- [Astroboard](https://github.com/wpplumber/astroboard) - Astroboard is an Umbraco dashboard to get insights of your contents, assets and members.
+- [Inox Tools: Request State](https://inox-tools.fryuni.dev/request-state) - Share request state between components both and from server to client.
+- [Inox Tools: Request Scoped Nano Stores](https://inox-tools.fryuni.dev/request-nanostores) - Enable Nano Stores to be used on the server and retain their server value on the client.
+- [Inox Tools: Content Git Authors](https://inox-tools.fryuni.dev/content-utils/git#getentrygitinfo) - Retrieve Git authors and co-authors involved in writing content collection entries.
+
+
+### Themes & Templates
+
+Several new Astro themes were added to the [Astro theme directory](https://astro.build/themes/) this month. Try them out!
+
+<ThemeGrid date="2024-08-01" />
+
+Also check out:
+
+[`simple-blog-with-tina`](https://github.com/Klrfl/simple-blog-with-tina) - An Astro + TinaCMS demo with live editing
+
+### Websites
+
+{/* 
+Sites of the month: UPDATE THESE
```

Comment:
Reminder to update!

Reply:
I decided not to show any sites of the month! I got lazy. 😅